src/middlewared/middlewared/plugins/enclosure_/map.py

```diff
@@ -120,11 +140,15 @@ async def _map_enclosures(self, enclosures, slots):
             elements.append(element)
 
         info = await self.middleware.call("system.info")
+        if '-MINI-' in ["system_product"]:
```

Comment:
This is wrong and will always evaluate to `False`

Reply:
Also, is there a jira ticket for this?

Reply:
I was gonna ask some to add jira label as I'm not in truenas org

Reply:
Ahh okay sounds good, I'm going to close the PR against `truenas/12.0-stable` since I can just add the `backport-12.0` label as well which will handle it for you.